### ISSUE TITLE
Shovel  - Problem with self-signed certificates

### ISSUE THREAD

**Issue by milokmet** [cloudamqp/lavinmq#741]
There is a mismatch between the documentation and the latest version of the code regarding to shovel and SSL.

In documentation there is staten that:

> URI: connection string to the source and destination clusters. The URI used to connect can be in any of the formats: 
> ...
> amqps://user:password@server-name?cacertfile=/path/to/cacert.pem&certfile=/path/to/cert.pem&keyfile=/path/to/key.pem&verify=verify_peer
> ...
> amqps://server-name?cacertfile=/path/to/cacert.pem&certfile=/path/to/cert.pem&keyfile=/path/to/key.pem&verify=verify_peer&fail_if_no_peer_cert=true&auth_mechanism=external 

However, when I specify source or destination URI in showel with cacertfile, I got this error in logs:
```
shovel[name: "kotlina-meter", vhost: "/"] Invalid parameter: cacertfile
Invalid parameter: cacertfile (ArgumentError)
   from usr/share/crystal/src/array.cr:1111:5 in 'new'
   from usr/src/lavinmq_2.0.0-rc.3/src/lavinmq/shovel/shovel.cr:75:17 in '->'
   from usr/share/crystal/src/fiber.cr:143:11 in 'run'
   from ???
```

Don't know how to speficy the src or dest URI to accept self signed certificates.

Thanks

**Comment by carlhoerberg** [comment]
You are right, I'm working on a fix.

**Comment by spuun** [comment]
Fixed in #45

**Comment by walro** [comment]
> Fixed in #45

Wrong 45, this 45: https://github.com/cloudamqp/amqp-client.cr/pull/45 😄

**Comment by spuun** [comment]
> > Fixed in #45
> 
> Wrong 45, this 45: [cloudamqp/amqp-client.cr#45](https://github.com/cloudamqp/amqp-client.cr/pull/45) 😄

🤦
